src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,283 @@
+---
+part_number: I
+chapter_number: 3
+title: Markup
+description: Markup chapter of the 2019 Web Almanac covering elements used, custom elements, value, products, and common use cases.
+authors: [bkardell]
+reviewers: [zcorpan, tomhodgins, matthewp]
+translators: [SilentJMA]
+discuss: 1758
+published: 2019-11-27T00:00:00.000Z
+last_updated: 2019-11-27T00:00:00.000Z
+---
+
+## Introduction
+
+En 2005, Ian "Hixie" Hickson a publié [ une analyse des données de balisage ](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) à partir de différents travaux précédents. Une grande partie de ce travail visait à examiner les noms de classes pour voir s’il existait une sémantique informelle commune adoptée par les développeurs et sur laquelle il serait peut-être logique de normaliser. Certaines de ses recherches ont permis d’informer de nouveaux éléments en HTML5.
```

Comment:
Still seeing random spaces being added to the markup. For example:

```suggestion
En 2005, Ian "Hixie" Hickson a publié [une analyse des données de balisage](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) à partir de différents travaux précédents. Une grande partie de ce travail visait à examiner les noms de classes pour voir s’il existait une sémantique informelle commune adoptée par les développeurs et sur laquelle il serait peut-être logique de normaliser. Certaines de ses recherches ont permis d’informer de nouveaux éléments en HTML5.
```

Reply:
done